src/main/java/com/danum/danum/service/board/question/QuestionServiceImpl.java

```diff
@@ -83,4 +87,29 @@ private void viewCheck(Question question, String email) {
         questionEmailRepository.save(token);
     }
 
+    @Override
+    public void likeStatus(Long id, String email) {
+        Question question = questionRepository.findById(id)
+                .orElseThrow(() -> new BoardException(ErrorCode.BOARD_NOT_FOUND_EXCEPTION));
+        Member member = memberRepository.findById(email)
+                .orElseThrow(() -> new MemberException(ErrorCode.MEMBER_NOT_FOUND_EXCEPTION));
+
+        Optional<QuestionLike> optionalQuestionLike = questionLikeRepository.findByQuestionIdAndMemberEmail(question, member);
+
+        if (optionalQuestionLike.isPresent()) {
+            question.minLike();
```

Comment:
마찬가지로 minLike보다 subLike 처럼 사용하면 좋을 것 같습니다.
```suggestion
            question.subLike();
```